### PR TITLE
Alborz UI

### PR DESCRIPTION
**Pull request by alborzk**

### CODE REVIEW

**Review by anandp1 — COMMENTED** [review]
clean up your code buddy u got buncha commented out code

File: src/components/home/movie-row.tsx [hunk 1]
```diff
@@ -25,37 +32,74 @@ const MovieRow: React.FC<MovieRowProps> = ({
...
+          username={username}
+          showSearch={showSearch}
+          setShowSearch={setShowSearch}
+          mutateUserData={mutateUserData}
+        />
+      )}
+
+      <div className="grid grid-cols-12 my-6 h-full">
+        <div></div>
+        <div></div>
```
Comment (anandp1) [inline src/components/home/movie-row.tsx]: why do u have two empty tags?

File: src/components/home/movie-row.tsx [hunk 2]
```diff
@@ -25,37 +32,74 @@ const MovieRow: React.FC<MovieRowProps> = ({
...
         </div>
-        <div className="group relative hidden sm:block h-full">
+        <div className="group relative hidden sm:block h-full px-2">
           <ChevronRightIcon
             onClick={slideRight}
-            className="absolute top-0 bottom-0 right-[0.4px] z-40 m-auto w-12 cursor-pointer opacity-0 text-white group-hover:bg-black group-hover:opacity-100 group-hover:bg-opacity-40 h-full"
+            className="absolute top-0 bottom-0 left-12 m-auto w-10 h-10 cursor-pointer opacity-10 hover:opacity-80 text-white"
           />
         </div>
+        <div></div>
```
Comment (anandp1) [inline src/components/home/movie-row.tsx]: same here

File: src/components/shared/navigation.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+              </li>
+            </ul>
+          </div>
+        {/* </Fade> */}
+        </div>
+      </nav>
+    </>
+  );
+};
+  export { Navigation };
```
Comment (anandp1) [inline src/components/shared/navigation.tsx]:
```suggestion

  export { Navigation };
  
```

**Review by anandp1 — COMMENTED** [review]
good job, few questions 😄

**Comment by jonathanplacatka** [comment]
this guy really added emojis to the readme 💀

File: src/components/home/movie-row.tsx (hunk 1 above)
Comment (jonathanplacatka) [reply src/components/home/movie-row.tsx]: u should clean your code before making a PR bud

**Review by anandp1 — COMMENTED** [review]
good job u can merge it after I fix up ur div tags

**Review by anandp1 — DISMISSED** [review]

**Pull request merged**